python_modules/libraries/dagster-polars/setup.py

```diff
@@ -38,13 +38,13 @@ def get_version() -> str:
     python_requires=">=3.8,<3.13",
     install_requires=[
         f"dagster{pin}",
-        "polars>=0.20.0,<1.3.0",
+        "polars>=0.20.0",
         "pyarrow>=8.0.0",
         "typing-extensions>=4.7.0",
         "universal_pathlib>=0.1.4",
     ],
     extras_require={
-        "deltalake": ["deltalake>=0.15.0"],
+        "deltalake": ["deltalake>=0.15.0,<0.19.0"],
```

Comment:
test failures that motivated pin:

```

[2024-08-16T16:18:46Z] FAILED dagster_polars_tests/test_polars_delta.py::test_polars_delta_native_partitioning - ValueError: Partition filters can only be used with PyArrow engine, use predicate instead. PyArrow engine will be deprecated in 1.0
--
  | [2024-08-16T16:18:46Z] FAILED dagster_polars_tests/test_polars_delta.py::test_polars_delta_native_multi_partitions - ValueError: Partition filters can only be used with PyArrow engine, use predicate instead. PyArrow engine will be deprecated in 1.0
  | [2024-08-16T16:18:46Z] FAILED dagster_polars_tests/test_polars_delta.py::test_polars_delta_native_partitioning_loading_single_partition - ValueError: Partition filters can only be used with PyArrow engine, use predicate instead. PyArrow engine will be deprecated in 1.0


```

Reply:
i believe this is what is listed in the breaking changes section here:
https://github.com/delta-io/delta-rs/releases/tag/python-v0.19.0